<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MDialog</class>
 <widget class="QDialog" name="MDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="WordTester.qrc">
    <normaloff>:/images/boss.ico</normaloff>:/images/boss.ico</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="WTable" name="tableWidget">
       <property name="minimumSize">
        <size>
         <width>261</width>
         <height>241</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="addButton">
         <property name="toolTip">
          <string>Add a new word</string>
         </property>
         <property name="text">
          <string>&amp;Add</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="resetButton">
         <property name="toolTip">
          <string>Reset all statuses to 'No-Test'</string>
         </property>
         <property name="text">
          <string>&amp;Reset</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="openButton">
       <property name="text">
        <string>Open List</string>
       </property>
       <property name="icon">
        <iconset resource="WordTester.qrc">
         <normaloff>:/images/folderopen.ico</normaloff>:/images/folderopen.ico</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="radio1">
          <property name="text">
           <string>Shuffle</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radio2">
          <property name="text">
           <string>In turn</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="testButton">
       <property name="text">
        <string>&amp;Test Words</string>
       </property>
       <property name="icon">
        <iconset resource="WordTester.qrc">
         <normaloff>:/images/boss.ico</normaloff>:/images/boss.ico</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>&amp;Save List</string>
       </property>
       <property name="icon">
        <iconset resource="WordTester.qrc">
         <normaloff>:/images/savemoney.ico</normaloff>:/images/savemoney.ico</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="kanboxExecLabel">
       <property name="text">
        <string>&lt;a href=&quot;Kanbox_Plugin&quot;&gt;Save to NetDisk&lt;/a&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Exit App</string>
       </property>
       <property name="icon">
        <iconset resource="WordTester.qrc">
         <normaloff>:/images/Stop.ico</normaloff>:/images/Stop.ico</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="versionLabel">
       <property name="text">
        <string>v3.45 by Liazm</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WTable</class>
   <extends>QTableWidget</extends>
   <header>wtable.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="WordTester.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>MDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>375</x>
     <y>302</y>
    </hint>
    <hint type="destinationlabel">
     <x>410</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
